crates/burn-core/Cargo.toml

```diff
@@ -130,6 +130,7 @@ burn-autodiff = { path = "../burn-autodiff", version = "0.18.0" }
 burn-dataset = { path = "../burn-dataset", version = "0.18.0", features = [
     "fake",
 ] }
+rstest = "0.25.0"
```

Comment:
we already have this in the workspace's dependency, so you can replace the version:

rstest = { workspace = true }